test/integration/applications-inventory/analysis/windup_basic_test.go

```diff
@@ -0,0 +1,87 @@
+package analysis
+
+import (
+	"encoding/json"
+	"testing"
+	"time"
+
+	"github.com/konveyor/tackle2-hub/api"
+	"github.com/konveyor/tackle2-hub/test/api/application"
+	"github.com/konveyor/tackle2-hub/test/assert"
+)
+
+//
+// Test application analysis
+// "Basic" means that there no other dependencies than the application itself (no need prepare credentials, proxy, etc)
+func TestBasicAnalysis(t *testing.T) {
```

Comment:
I partially agree, but having this in test/integration directory (instead of test/api), expecting its move to different repo (or at least be just the integration test proving that Hub can triger addon analysis) and having more advanced analysis test in different repos looked OK-ish for me. Also it could be a helpful example for other&QE how to use the test methods from Hub.
Fot that reason I put it to this PR. If this would block this PR from getting in, I can remove it.